command/volume_snapshot_list.go

```diff
@@ -148,14 +148,10 @@ func (c *VolumeSnapshotListCommand) Run(args []string) int {
 
 func csiFormatSnapshots(snapshots []*api.CSISnapshot, verbose bool) string {
 	rows := []string{"Snapshot ID|Volume ID|Size|Create Time|Ready?"}
-	length := 12
-	if verbose {
-		length = 30
-	}
 	for _, v := range snapshots {
 		rows = append(rows, fmt.Sprintf("%s|%s|%s|%s|%v",
-			limit(v.ID, length),
-			limit(v.ExternalSourceVolumeID, length),
```

Comment:
The external volume ID can be _extremely_ long for some cloud providers (ex. Azure) and I don't think it's required for Nomad CLI operations the way the snapshot ID is. It might be worth having the limit removed by the `-verbose` flag instead of by default for this field.

Reply:
Makes sense, I've updated this to still shorten the `ExternalSourceVolumeID`.